### PR TITLE
[Merged by Bors] - Create PhaestusFox

### PR DESCRIPTION
**Pull request by PhaestusFox**

### CODE REVIEW

File: Community Members/PhaestusFox [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# File name of your profile picture if you uploaded one
+# If you provide a file, be mindful of its size
+# If you also add your github profile, you can use "GitHub" to use your GitHub profile picture
+# This can be any image that follows our code of conduct.
+# Optional
+profile-picture = "GitHub"
+
+# Short presentation of yourself. This has a limit of 180 characters.
+# Optional
+bio = "I am a small youtuber, attemted game dev and avid doc reader. I'm happy to help where ever and when ever I can"
```
Comment (alice-i-cecile) [inline Community Members/PhaestusFox:14]:
```suggestion
bio = "I am a small youtuber, attempted game dev and avid doc reader. I'm happy to help whereever and whenever I can"
```

There were 3 typos here :)

**Review by alice-i-cecile — APPROVED** [review]

**Comment by mockersf** [comment]
the file should have a `.toml` extension

**Pull request closed without merging**